accounts/accounts/routes/ui.py

```diff
@@ -192,3 +192,9 @@ def captcha() -> Response:
     token = request.args.get('token')
     data, code, headers = captcha_image.get(token, secret, request.remote_addr, font)
     return send_file(data['image'], mimetype=data['mimetype']), code, headers
+
+
+@blueprint.route('/auth_status', methods=['GET'])
+def auth_status() -> Response:
+    # TODO Test resurces like DB and redis
+    return make_response("OK")
```

Comment:
Is there a ticket for this future TODO work?

Reply:
Yes, I've found it nice to check not just that the service is running but also that any resources the service is dependent on are available and working.